### ISSUE TITLE
[Accessibility][Headings and Labels - Flow Chart] : The name of the flow chart should be defined under heading tag. 

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2028]
**User Experience** 
Users who rely on AT will face difficulty and easily confused if the headings are not defined correctly.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE.
2. Select 'Design Flow' from left navigation.
3. Now, turn on the narrator and navigate with 'H' i.e. shortcut for Headings.
4. Observe the issue in the flow chart area.

**Actual:** 
The heading of the flow chart i.e. 'Dialog started' is not defined under heading tags.

**Expected:**
The name of the flow chart should be defined under heading tag. So, that the user will able to navigate quickly through the heading shortcut.

**MAS Impacted:** [MAS2.4.6](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={0cf80872-800d-4643-90cf-0c1c3d3e6260})

**Attachment:**
![Heading is not defined for the flow chart](https://user-images.githubusercontent.com/60913131/74919441-9bd67100-53f0-11ea-9a67-c130d45318a2.png)

**Comment by ashish315** [comment]
@corinagum As checked, now it is defined in heading {specifically in h4} but the heading structure is not logically correct as no h1, h2 or h3 is defined on the page.

**Comment by corinagum** [comment]
@ashish315 thank you verifying. I have moved this back to the To Do column.

**Comment by ashish315** [comment]
As checked and verified on below environment, the issue is fixed now. So, we are closing the issue.

**Test Environment**
Browser: Microsoft Edge Dev {Version 83.0.467.0 (Official build) dev (64-bit)}
OS build: 2004 (19588.1000)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)
